packages/shared/src/components/search/SearchResults/SearchResultsSources.tsx

```diff
@@ -0,0 +1,45 @@
+import React, { ReactElement } from 'react';
+import { Source } from '../../../graphql/sources';
+import { WidgetCard } from '../../widgets/WidgetCard';
+import { UserHighlight, UserType } from '../../widgets/PostUsersHighlights';
+import { ListItemPlaceholder } from '../../widgets/ListItemPlaceholder';
+
+interface SearchResultsSourcesProps {
+  items: Source[];
+  isLoading: boolean;
+  onSourceClick: (source: Source) => void;
+}
+
+export const SearchResultsSources = (
+  props: SearchResultsSourcesProps,
+): ReactElement => {
+  const { items, isLoading, onSourceClick } = props;
+  return (
+    <WidgetCard heading="Related sources">
+      {!!items.length && (
```

Comment:
Same here, if there is a possibility for the value to be `null`.
```suggestion
      {!!items?.length && (
```